impala/hiveserver2.py

```diff
@@ -758,39 +759,47 @@ def threaded(func):
 
 def connect(host, port, timeout=None, use_ssl=False, ca_cert=None,
             user=None, password=None, kerberos_service_name='impala',
-            auth_mechanism=None, krb_host=None):
+            auth_mechanism=None, krb_host=None, use_http_transport=True,
+            http_path=''):
     log.debug('Connecting to HiveServer2 %s:%s with %s authentication '
               'mechanism', host, port, auth_mechanism)
-    sock = get_socket(host, port, use_ssl, ca_cert)
 
-    if krb_host:
-        kerberos_host = krb_host
+    if use_http_transport:
+        transport = get_http_transport(host, port, http_path=http_path,
```

Comment:
Validate that we don't have kerberos enabled for http?

Reply:
Done